src/entity/User.ts

```diff
@@ -42,23 +42,25 @@ export class User {
   reputation: number;
 
   @Column({ length: 39, nullable: true })
-  @Index()
+  @Index('users_username_unique', { unique: true })
   username?: string;
 
   @Column({ type: 'text', nullable: true })
   bio?: string;
 
   @Column({ length: 15, nullable: true })
-  @Index()
+  @Index('users_twitter_unique', { unique: true })
```

Comment:
Why are you changing it to unique?
If so, we need to validate if there can be two users with twitter as null. Because null sometimes is also treated as unique. Can you verify please?

Reply:
In our previous validation in the gateway, we leveraged the uniqueness of the column.
Yes, I've read in postgresql docs being null in multiple rows is allowed. Sample test below:
![image](https://user-images.githubusercontent.com/13744167/185052653-2d10fab2-29a8-413e-b8b3-9a47dc6640ff.png)